unilog.go

```diff
@@ -334,10 +343,12 @@ func (u *Unilog) Main() {
 
 	fileName := u.target
 
-	Stats, _ = statsd.New(StatsdAddress)
+	Stats, _ = statsd.New(u.StatsdAddress)
 
 	Stats.Tags = append(Stats.Tags, fmt.Sprintf("FileName:%s", fileName))
-	Stats.Tags = append(Stats.Tags, strings.Split(statstags, ",")...)
```

Comment:
It does, but `append` makes `Stats.Tags` length 2 (instead of 1, which is what it should be if `statstags` is empty), which adds a dangling comma at the end of the tags.